lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9058,12 +9058,26 @@ class SwiftifyInfoPrinter {
   }
 
   void printAvailability() {
+    if (!hasMacroParameter("spanAvailability"))
+      return;
     printSeparator();
     out << "spanAvailability: ";
     printAvailabilityOfType("Span");
   }
 
 private:
+  bool hasMacroParameter(StringRef ParamName) {
+    ValueDecl *D = getDecl("_SwiftifyImport");
+    auto *SwiftifyImportDecl = dyn_cast_or_null<MacroDecl>(D);
+    assert(SwiftifyImportDecl);
+    if (!SwiftifyImportDecl)
+      return false;
+    for (auto *Param : *SwiftifyImportDecl->parameterList)
```

Comment:
Could this be replaced with `llvm::find_if` or something similar?

Reply:
It could be replaced with `any_of`, but it wouldn't really be any easier to read imo (nor more concise), and is a bit more annoying to debug with the jumping between stack frames